### ISSUE TITLE
vCard labels standardisierter Vor-/Name

### ISSUE THREAD

**Issue by flo-so** [corona-warn-app/cwa-quick-test-frontend#191]
For these 2 fields, red marked on the Screenshot, there are no labels defined in vCard-Spec. The documentation for the QR-Code is also missing in the documents.
What are the correct vCard-labels to fill those two fields via QR-Code-Scan?
![Screenshot_20210706-102238_Firefox](https://user-images.githubusercontent.com/84222822/124583316-33f0c100-de53-11eb-9b2e-b31306343bf7.jpg)

**Comment by vaubaehn** [comment]
Hi @flo-so , is it still necessary today to enter the standardised name manually?
If yes, I consider this a big problem at the current point of the whole project, that would need to be addressed soon.
Thanks for your response.

**Comment by flo-so** [comment]
@vaubaehn 
If it should automatically fill these fields with the data from Vorname and Name, this is a bug.
It neither fills on manual enter nor via QR-Code.
(On Test System)

**Comment by vaubaehn** [comment]
@flo-so 
I beg your pardon if I introduced some confusion. You confirmed that the standardised names are not generated automatically. After I asked you above (before your response), I tried to find hints that this function was already implemented, but no.

I see this as a problem, as the Corona-Warn-App relies on names to match certificates for a single person. In daily practice, the regular name already causes often problems due to accidentally superflous spaces, wrong capitalisation, wrong entries of non-latin characters.
Using standardised names can prevent these flaws, but only if automatically derived from the regular names. They can be used by wallet apps for matching different certificates to one person. Verifier apps in some countries may use standardised names in advance of regular name, to avoid confusion e.g. with Cyrillic letters.

Forcing testing staff (or staff issuing vaccination certificates) to also enter standardised names manually introduces another source of error, and causes more strain. In worst case the verification process (e.g. at border control) may get troubled.

Coding an adequate algorithm for deriving standardised names could be done in an hour task, maybe?

@ascheibal was already pointing to a python script that does one main part (but without converting to uppercase, eliminating superfluous spaces and replacing spaces with '<', like also demanded in the hcert spec...)

I'll create a separate issue for that soon.
In best case, your issue here then also gets obsolete after implemented.
